chroma-manager/requirements.dev

```diff
@@ -18,4 +18,4 @@ pyflakes
 pygraphviz
 sphinx==1.1.3
 werkzeug
-iml-common
+iml-common>=1.0,<1.1
```

Comment:
I think we should remove `requirements.dev` and only use a single set of rpm sourced deps.

Reply:
I agree, given we go to the effort of creating a set of rpm deps, might as well use them for all testing

Reply:
Ahhh.  Wait.  This dep is in the wrong place then!  It probably needs to go into https://github.com/intel-hpdd/intel-manager-for-lustre/blob/pin-iml-common/chroma-dependencies/Makefile.  The caveat being that that whole `chroma-dependencies/` subdir needs to go away entirely and the `requirements.txt` building that it assists in refactored to somewhere/thing else.

But that's where it is and how it is done currently, so that is where this needs to go.

Reply:
> I think we should remove requirements.dev

So, these are (supposed to be) the deps that a "developer" would want (i.e. `pip`) installed in his "local" environment for doing development/testing and/or for "local source pool" type CI tests.

> and only use a single set of rpm sourced deps.

How does that work for developers on Windows and/or MacOS?

Reply:
They would use the VM setup for dev which many of us are already doing.